Ui/Component/Listing/Column/Stores.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Snowdog\Menu\Ui\Component\Listing\Column;
+
+use Magento\Store\Ui\Component\Listing\Column\Store\Options as StoreOptions;
+
+class Stores extends StoreOptions
```

Comment:
are you sure there is no method for that in magento core? if I understand it correctly you want to add `All Store Views` option, right?

Reply:
For the previous view I don't think there was such method. Now that this field is changed to `All Store Views` style, there is a core Magento method so I removed this file and use the core one.